tron/core/job.py

```diff
@@ -123,7 +123,17 @@ def __init__(
         log.info(f"{self} created")
 
     @staticmethod
-    def to_json(state_data: dict) -> Optional[str]:
+    def from_json(state_data: str):
+        """deserialize the JSON string to python objects."""
+        # We store the following fields for jobs in DynamoDB: enabled and list of run numbers
+        try:
+            return json.loads(state_data)
```

Comment:
same as further above: it'd probably be a good idea to make this `from_json` match the other explicit ones (since we're more likely to touch the `Job` class than the case above)